package.json

```diff
@@ -9,13 +9,15 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.26.0",
+    "node-texturemapper": "https://github.com/Epitrochoid/node-texturemapper.git",
```

Comment:
I take it we are going to pilot this package before it gets an official published version?